properties/base.py

```diff
@@ -155,10 +155,15 @@ def __init__(self, **kwargs):
                                'or attribute'.format(key))
             setattr(self, key, kwargs[key])
 
-    def _get(self, name):
+    def _get(self, name, default=None):
         if name in self._backend:
             return self._backend[name]
 
+        if default is utils.undefined:
```

Comment:
This module uses utils.undefined as well as basic.undefined. It's the same thing but consistency would be great.